sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/ConfidentialLedgerClientLiveTests.cs

```diff
@@ -447,6 +428,41 @@ public async Task CustomRoleTest()
         }
         #endregion
 
```

Comment:
Can we add a really simple test for tags feature?  Just test if we do a write with tags, you can recover it?